docs/src/pages/components/timeline/timeline.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Timeline React component
+components: Timeline, TimelineItem, TimelineSeparator, TimelineDot, TimelineConnector, TimelineContent, TimelineOppositeContent
+---
+
+# Timeline
+
+<p class="description">The timeline displays a list of events in chronological order.</p>
+
+**Note:** This component is not documented in the [Material Design guidelines](https://material.io/), but Material-UI supports it.
+
+## Simple timeline
+
+Simple timeline showing list of events.
+
+{{"demo": "pages/components/timeline/SimpleTimeline.js"}}
+
+## Right aligned timeline
+
+The timeline can be positioned on the right side of the events.
+
+{{"demo": "pages/components/timeline/RightAlignedTimeline.js"}}
+
+## Alternating timeline
+
+The timeline can display the events on alternating sides.
+
+{{"demo": "pages/components/timeline/AlternateTimeline.js"}}
+
+## Color
+
+The `TimelineDot` can appear in different colors.
+
+{{"demo": "pages/components/timeline/ColorsTimeline.js"}}
+
+## Outlined
+
+{{"demo": "pages/components/timeline/OutlinedTimeline.js"}}
+
+## Opposite content
+
+The timeline can contain an element opposite of the content.
```

Comment:
Hmm, after some thought was is your opinion on changing this to something like:

```suggestion
The timeline can display content on opposite sides.
```

cc @mbrookes